code/modules/vending/_vending.dm

```diff
@@ -1340,6 +1340,8 @@ GLOBAL_LIST_EMPTY(vending_products)
 	add_overlay(panel_type)
 	//and references the deity
 	name = "[GLOB.deity]'s Consecrated Vendor"
+	max_integrity = 700
+	max_loaded_items = 40
```

Comment:
Why on initialize instead of on declaration?

Reply:
Moved, my mistake